src/reducers/sort.reducer.spec.js

```diff
@@ -0,0 +1,28 @@
+import sortReducer from './sort';
+import constants from '../constants';
+
+describe('[ Reducers ] Sort', () => {
+  let state;
+  beforeEach(() => { state = constants.NO_SORT; });
```

Comment:
You should define constant variables once (outside of the beforeEach). Use beforeEach when you want to have clean data for each test case. In this case we have common variable wich we re not going to change